andela/travel_tool_front
# #160833565 Notifications for rejected requests

#### What does this PR do?
* This PR displays notifications for rejected requests

#### Description of Task to be completed?
- Display notifications for rejected requests
-  Update the `Notification Pane` to show `view details` for rejected notifications

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-rejected-request-notification-160833565`
- Install dependencies and start the server - `make start`
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/75)
- Create a new travel request
- Click on a request after routing to `My Approvals` page and reject the request
- A notification will be sent to you.
- Click on the `Bell icon on the navbar` to see the new notification

#### Any background context you want to provide?
Ensure to log out of the app and log back in after setting up the server using this [guide](https://github.com/andela/travel_tool_back/pull/75).

#### What are the relevant pivotal tracker stories?
* [#160833565](https://www.pivotaltracker.com/story/show/160833565)

#### Screenshots (if appropriate)
<img width="1439" alt="screen shot 2018-09-28 at 4 49 29 pm" src="https://user-images.githubusercontent.com/27569442/46221490-4609b080-c345-11e8-87e0-94287323b608.png">

#### Questions:
* N/A
